python_files/supplementary_plot_helpers.py

```diff
@@ -786,14 +812,27 @@ def run_cancer_mask_inclusion_tests(
     # cell_boundary_max_hole_sizes = [int(tm * max_hole_size) for tm in threshold_mults]
     cell_boundary_border_sizes = [int(tm * base_border_size) for tm in threshold_mults]
 
-    cell_boundary_sigma_data = {s: [] for s in cell_boundary_sigmas}
-    cell_boundary_channel_thresh_data = {ct: [] for ct in cell_boundary_channel_threshes}
-    cell_boundary_min_mask_size_data = {mms: [] for mms in cell_boundary_min_mask_sizes}
-    # cell_boundary_max_hole_size_data = {mhs: [] for mhs in cell_boundary_max_hole_sizes}
-    cell_boundary_border_size_data = {bs: [] for bs in cell_boundary_border_sizes}
+    if os.path.exists(pathlib.Path(save_dir) / "intermediate_data.json"):
```

Comment:
Looks like there's a bunch of changes to the feature extraction pipeline in here too?

Reply:
Mb, was testing something related to cancer cell boundaries on this PR and forgot to remove the code.